connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -262,6 +269,9 @@ protected static ConfigDef baseConfigDef() {
                 .define(REST_ADVERTISED_HOST_NAME_CONFIG, Type.STRING,  null, Importance.LOW, REST_ADVERTISED_HOST_NAME_DOC)
                 .define(REST_ADVERTISED_PORT_CONFIG, Type.INT,  null, Importance.LOW, REST_ADVERTISED_PORT_DOC)
                 .define(REST_ADVERTISED_LISTENER_CONFIG, Type.STRING,  null, Importance.LOW, REST_ADVERTISED_LISTENER_DOC)
+                .define(REST_REQUEST_MAX_TIMEOUT_MS_CONFIG, Type.LONG,
+                        REST_REQUEST_MAX_TIMEOUT_MS_DEFAULT, Range.atLeast(DEFAULT_REST_REQUEST_TIMEOUT_MS),
```

Comment:
The alternative here would be to have no validator and allow setting this config to values lower than 90 seconds (default request timeout). In this case, we would also need to alter the default timeout used by `ConnectorsResource` and `ConnectorPluginsResource` because this config is the "max request timeout". However, we would probably not want to allow tweaking the default timeout to be greater than 90 seconds via this config, because that's what the `timeout` query parameter is for and the interaction between this config and the query param could be confusing.